cli/integration_tests/setup_git.sh

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env bash
 
 TARGET_DIR=$1
+SKIP_INSTALL=$2
 git init ${TARGET_DIR} --quiet --initial-branch=main
 GIT_ARGS="--git-dir=${TARGET_DIR}/.git --work-tree=${TARGET_DIR}"
 git ${GIT_ARGS} config user.email "turbo-test@example.com"
 git ${GIT_ARGS} config user.name "Turbo Test"
 echo "script-shell=$(which bash)" > ${TARGET_DIR}/.npmrc
-npm --prefix=${TARGET_DIR} install --silent
+# Some examples don't use npm
+if [[ "$SKIP_INSTALL" != "--skip-install" ]]; then
+  npm --prefix=${TARGET_DIR} install --silent
+fi
```

Comment:
similar thing in #4190 (which i still haven't gotten passing), fyi https://github.com/vercel/turbo/pull/4190/files#diff-b7727a8fd178a7b4189de2fd392fa25a3f07140042eafbb2e339fa3407c63e01R14-R16

we can use this, it's fine